platform/kubernetes/kubernetes.go

```diff
@@ -115,8 +115,8 @@ func createHTTPClient(caCert []byte, insecureTLS bool) *http.Client {
 		DualStack: true,
 	}
 	// Copy from the definition of http.DefaultTransport.
+	// Don't use Proxy.
```

Comment:
Why not `Proxy: nil` here, like the other places?

Reply:
Since the zero value of the Transport.Proxy is nil, it does not explicitly set.